src/HL7/Segments/OBX.php

```diff
@@ -207,4 +217,15 @@ public function getObservationMethod(int $position = 17)
     {
         return $this->getField($position);
     }
+    
+    public function getEquipmentInstanceIdentifier(int $position = 18)
+    {
+        return $this->getField($position);
+    }
+
+    public function getDateTimeOfAnalysis(int $position = 19)
+    {
+        return $this->getField($position);
+    }
```

Comment:
Since these fields belong to HL7 versions later than 2.3, the reference URL in the class doc doesn't have it. Can you please update the reference URL at the top of OBX.php to https://hl7-definition.caristix.com/v2/HL7v2.5.1/Segments/OBX 

(you should be able to edit the file on this page directly, or you can edit locally on the branch and push it)